src/components/LayerCreator.svelte

```diff
@@ -1,14 +1,22 @@
 <script context="module" lang="ts">
-	import GeoJsonLayer from './GeoJsonLayer.svelte';
+    import GeoJsonLayer from './GeoJsonLayer.svelte';
+    import GeotiffLayer from './GeotiffLayer.svelte';
```

Comment:
I might rename this `GeoTiffLayer` for consistency with `GeoJsonLayer`. Thoughts?

Reply:
Sure thats fine